src/bomberman/entity/staticobjects/BlackBomb.java

```diff
@@ -32,6 +45,29 @@ public BlackBomb(int x, int y) {
         bomb_animations=new BombAnimations(this);
         sprite=bomb_animations.getBlackBomb();
         entityBoundary = new RectBoundedBox(positionX, positionY, width, height);
+        addedDate=new Date();
+        bombState=STATE.ACTIVE;
+    }
+    
+    public boolean isAlive(){
+        STATE s = checkBombState();
+        if(s==STATE.DEAD){
+            return false;
+        }
+        else{
+            if(s==STATE.ACTIVE||s==STATE.INACTIVE){

```

Comment:
Is this if statement needed? After the else on line 57 the method is going to return true whether or not the if statement on line 58 is true or false.

Reply:
yeah not needed, I was doing something else and forgot to remove it. Sorry 😆